dev/VSIX/Extension/Cs/Dev17/WindowsAppSDK.Cs.Extension.Dev17.csproj

```diff
@@ -48,19 +47,19 @@
     <Content Include="$(RepoRoot)LICENSE">
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
-    <Content Include="ExtensionPack.vsext">
+    <Content Condition="'$(Deployment)'=='Standalone'" Include="ExtensionPack.vsext">
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
```

Comment:
Are we OK with this extension being included in both the Component and Standalone cases in the Dev16 cases?

Reply:
We don't build a Component extension for Dev16, and it's very unlikely we ever will. But it's simple enough to add for future-proofing.